modules/persistent-storage-manila-usage.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-manila.adoc
+
+[id="persistent-storage-manila-usage-{context}"]
+= Provisioning an OpenStack Manila persistent volume
+
+OpenStack Manila shares are dynamically provisioned as needed. When the
+PersistentVolumeClaim is deleted the provisioner will automatically 
+delete and unexport the OpenStack Manila share.
+
+.Prerequisites
+
+* The OpenStack Manila external provisioner must be installed.
+
+.Procedure
+
+* Create a PersistentVolumeClaim using the corresponding
+StorageClass
+
```

Comment:
Add a + here so that it's indented properly with "step 1"

Reply:
Corrected.